### PR TITLE
Custom order attachments

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

File: be/src/Project/Services/Ordering/Ordering.Api/Infrastructure/Orders/Entities/OrderAttachment.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.ComponentModel.DataAnnotations;
+
+namespace Ordering.Api.Infrastructure.Orders.Entities
+{
+    public class OrderAttachment : EntityMedia
+    {
+        [Required]
+        public Guid OrderId { get; set; }
+
+        internal IEnumerable<Guid> Select(Func<object, object> p)
```
Comment (dawiddworak88) [inline be/src/Project/Services/Ordering/Ordering.Api/Infrastructure/Orders/Entities/OrderAttachment.cs]: **suggeston:** to remove

File: be/src/Project/Services/Ordering/Ordering.Api/Services/OrdersService.cs [hunk 2]
```diff
@@ -105,25 +120,46 @@ public async Task CheckoutAsync(CheckoutBasketServiceModel serviceModel)
...
+                var attachments = new List<string>();
+
+                foreach (var attachmentId in serviceModel.Attachments.OrEmptyIfNull())
+                {
+                    var newAttachment = new OrderAttachment
+                    {
+                        OrderId = order.Id,
+                        MediaId = attachmentId
+                    };
+
```
Comment (dawiddworak88) [inline be/src/Project/Services/Ordering/Ordering.Api/Services/OrdersService.cs]: **suggestion (non-blocking):** redundant new line

File: fe/src/shared/components/Files/Files.js [hunk 3]
```diff
@@ -16,11 +16,10 @@ function Files(props) {
     };
 
     return (
-        
         <Fragment>
             {props.files &&
                 <section className="section files">
-                    <h3 className="title is-4">{props.downloadFilesLabel}</h3>
+                    <h3 className="title is-4">{props.isAttachments ? props.attachmentsLabel : props.downloadFilesLabel}</h3>
```
Comment (dawiddworak88) [inline fe/src/shared/components/Files/Files.js]: **suggestion:** I'd rather avoid making it attachment-specific

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

File: be/src/Project/Services/Ordering/Ordering.Api/Infrastructure/Orders/Entities/OrderAttachment.cs (hunk 1 above)
Comment (SebaKas21) [reply be/src/Project/Services/Ordering/Ordering.Api/Infrastructure/Orders/Entities/OrderAttachment.cs]: @dawiddworak88 Ups, I haven't seen it.

Now is fixed

File: fe/src/shared/components/Files/Files.js (hunk 3 above)
Comment (SebaKas21) [reply fe/src/shared/components/Files/Files.js]: @dawiddworak88 

we should recognize files or attachments, for me is useless create new component. 

Or my second think is about change downloadLabel "Pliki do pobrania" to "Załączone pliki" and delete props isAttachments

Comment (dawiddworak88) [reply fe/src/shared/components/Files/Files.js]: Hi @SebaKas21 , let's rename it to "Pliki" and remove all the "isAttachments" properties.

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

File: be/src/Project/Web/Seller/Seller.Web/Shared/ViewModels/FilesViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        public string DownloadLabel { get; set; }
+        public string CopyLinkLabel { get; set; }
+        public string FilenameLabel { get; set; }
+        public string NameLabel { get; set; }
+        public string DescriptionLabel { get; set; }
+        public string SizeLabel { get; set; }
+        public string LastModifiedDateLabel { get; set; }
+        public string CreatedDateLabel { get; set; }
+        public string AttachmentsLabel { get; set; }
+        public bool? IsAttachments { get; set; }
```
Comment (dawiddworak88) [inline be/src/Project/Web/Seller/Seller.Web/Shared/ViewModels/FilesViewModel.cs]: Hi @SebaKas21 , isn't it a leftover? Let's remove all of IsAttachments

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

**Review by dawiddworak88 — APPROVED** [review]

**Pull request merged**